packages/core/src/App/Containers/UrlUnavailableModal/__tests__/url-unavailable-modal.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import UrlUnavailableModal from '../url-unavailable-modal';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+describe('<UrlUnavailableModal />', () => {
+    let modalRootEl: HTMLDivElement;
+
+    beforeAll(() => {
+        modalRootEl = document.createElement('div');
+        modalRootEl.setAttribute('id', 'modal_root');
+        document.body.appendChild(modalRootEl);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modalRootEl);
+    });
+
+    const modalHeading = /The URL you requested/i;
+    const modalInfo = /This could be because:/i;
+    const okButtonName = 'OK';
+
+    const store = mockStore({
+        ui: { is_mobile: false, isUrlUnavailableModalVisible: true, toggleUrlUnavailableModal: jest.fn() },
```

Comment:
in p_ackages/stores/src/mockStore.ts_ `is_mobile` is `false` by default if I remember correctly. Please, ignore if it's not relevant.

Reply:
you're right, good catch :)